governance/program/src/state/proposal.rs

```diff
@@ -102,6 +102,15 @@ pub enum VoteType {
     },
 }
 
+/// Proposal type of definition the Proposal finalize vote processing
+#[derive(Clone, Debug, PartialEq, Eq, BorshDeserialize, BorshSerialize, BorshSchema)]
+pub enum ProposalType {
+    /// Standard proposal that verifies the defined threholds
+    VoteThreshold,
+    /// Threshold is not considered on finalizing votes, all options succeeds automatically
+    NoThresholdAllSucceed,
```

Comment:
@SebastianBor I see.

* please, can you elaborate what's are the key points that are required to be adjusted here (probably such kind of proposal may not permit transactions?)
* is the way how the `Proposal` struct is changed fine from your perspective or you consider that could be done better in code style?

Reply:
I think we don't need this `ProposalType`  enum at all because a proposal without `Deny` vote is already a survey only `Proposal` and can't have executable instructions.

For survey only proposals we should only consider the whole `Proposal` as `Succeeded` or `Failed` and the individual options should just stay with `OptionVoteResult`  as `None` because there is no `Succeeded `/`Failed` condition for them.
The `Succeeded`/`Failed` status of the whole proposal should be determined based on whether the relevant quorum was achieved and hence making the survey valid. For example if enough members participated with the `AttendanceQuorum` in the survey then we consider it as `Succeeded`.